Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -908,6 +919,9 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
                 }
                 completionHandler?(nil)
             }
+        case .setRegistrationKeyValidity(let validity):
+            setKeyValidity(validity.timeInterval)
+            completionHandler?(nil)
```

Comment:
Changing the key validity from our debug menu was no longer working.  These changes add that support back.

Reply:
Nice, thanks!